Makefile

```diff
@@ -0,0 +1,38 @@
+goimports := golang.org/x/tools/cmd/goimports@v0.1.12
+golangci_lint := github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
+
+.PHONY: test
+test:
+	@go test -v ./...
+
+golangci_lint_path := $(shell go env GOPATH)/bin/golangci-lint
+
+$(golangci_lint_path):
+	@go install $(golangci_lint)
+
+.PHONY: lint
+lint: $(golangci_lint_path)
+	@CGO_ENABLED=0 $(golangci_lint_path) run --timeout 5m
+
+.PHONY: format
+format:
+	@find . -type f -name '*.go' | xargs gofmt -s -w
+	@for f in `find . -name '*.go'`; do \
```

Comment:
Maybe a comment on what is this doing exactlu?

Reply:
what would that comment say that find isn't already saying?